playkit/src/main/java/com/kaltura/playkit/plugins/KalturaLiveStatsPlugin.java

```diff
@@ -235,16 +238,50 @@ private void sendLiveEvent(final long bufferTime) {
         // Parameters for the request -
         // String baseUrl, int partnerId, int eventType, int eventIndex, int bufferTime, int bitrate,
         // String startTime,  String entryId,  boolean isLive, String referrer
-        RequestBuilder requestBuilder = LiveStatsService.sendLiveStatsEvent(baseUrl, partnerId, isLive ? 1 : 2, eventIdx++, bufferTime,
-                lastReportedBitrate, sessionId, mediaConfig.getStartPosition(), entryId, isLive, PlayKitManager.CLIENT_TAG, "hls");
+        long distanceFromLive = 0;
+        if (player != null) {
+            distanceFromLive = player.getDuration() - player.getCurrentPosition();
+        }
+
+        RequestBuilder requestBuilder = LiveStatsService.sendLiveStatsEvent(pluginConfig.getBaseUrl(),
+                pluginConfig.getPartnerId(),
+                (distanceFromLive <= FIFTEEN_SEC) ? KLiveStatsEvent.LIVE.value : KLiveStatsEvent.DVR.value,
```

Comment:
this is something that already there
the Web implemented this this way:

this.bind( 'seeked seeking onpause', function() {
					if ( _this.getPlayer().isDVR() ) {
						_this.eventType = 2;
					}
				});